turbinia/lib/google_cloud.py

```diff
@@ -93,3 +98,44 @@ def setup_stackdriver_traceback(project_id):
     msg = 'Error enabling GCP Error Reporting: {0:s}'.format(str(exception))
     raise TurbiniaException(msg)
   return client
+
+
+def get_logs(output_dir, project_id, days, query=None):
+  """Copies stackdriver logs to a local directory.
+
+  Attributes:
+    project_id: The name of the Google Cloud project.
+    output_dir: The directory where logs are stored.
+    query: Query to use to pull stackdriver logs. 
+    days: number of days we want history for.
+  Raises:
+    TurbiniaException: When an error happens pulling the logs.
+  """
+  if not query:
+    query = 'jsonPayload.python_logger="turbinia"'
+  start_time = datetime.datetime.now() - timedelta(days=days)
```

Comment:
Should we set a default value for days in the method definition too just to be safe?

Reply:
Done.